jitbuilder/release/.gitignore

```diff
@@ -0,0 +1,33 @@
+###############################################################################
+#
+# (c) Copyright IBM Corp. 2016, 2016
+#
+#  This program and the accompanying materials are made available
+#  under the terms of the Eclipse Public License v1.0 and
+#  Apache License v2.0 which accompanies this distribution.
+#
+#      The Eclipse Public License is available at
+#      http://www.eclipse.org/legal/epl-v10.html
+#
+#      The Apache License v2.0 is available at
+#      http://www.opensource.org/licenses/apache2.0.php
+#
+# Contributors:
+#    Multiple authors (IBM Corp.) - initial implementation and documentation
+###############################################################################
+
+include/compiler/
```

Comment:
I created the new .gitignore file since other components did so, namely [the Test compiler](https://github.com/eclipse/omr/blob/master/fvtest/compilertest/tests/.gitignore) and [OMRRewriter](https://github.com/eclipse/omr/blob/master/tools/compiler/OMRRewriter/.gitignore). However, I didn't look at those files to realize they each only contained one entry.

I think there should be a wider discussion about this. I personally feel we should use multiple .gitignores as a way to keep components as separated as possible. However, the extra hassle may not be worth it if there are only a few component-specific files. For reference, there are around 70 component specific rules (including the ones in this PR).